draft-scenarios/coops/the_ambushed_accord.tex

```diff
@@ -0,0 +1,119 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative Scenario}{The Ambushed Accord}{\images/earth_magic.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} Wololoeren
+
+\textit{The war had dragged on far too long. Armies were shattered, lands burned, and hope had dwindled to barely a whisper. At last, the rival lords agreed to set aside their grievances and forge an Accord — a fragile peace born of desperation.\\
+Messengers rode beneath white banners, oaths were sworn, and for the first time in years, the clash of steel gave way to uneasy silence. Yet peace has its own enemies.\\
+As the allied hosts gathered in the valley of Laren's Cross, dawn broke with the sound of horns — not of welcome, but of war.\\
+Out of the mist, enemy armies struck with deadly precision. Someone had betrayed the Accord.\\Now, surrounded and outnumbered, the fate of your alliance lies in your hands. Stand together — or fall alone.}  % no-check-caps
+\subsection*{\MakeUppercase{Scenario Length}}
+14 Rounds
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2-6
+
+\textbf{Starting Resources:} 2 \svg{gold}, 3 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+
+\begin{itemize}
+  \item 2 × Search (2) the \svg{bronze} Neutral Deck and 1 x \svg{morale_positive}
+\end{itemize}
+
+\textbf{Town Buildings:} City Hall
+
+\textbf{Map Tile Pool:} Every player takes 2 random Far (II-III) Map Tiles and 2 random Near (IV-V) Map Tiles.
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles ($P$ stands for the number of players) and arrange them as shown in the Scenario map layout:
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tiles
+  \item 1 × Center (VI–VII) Map Tile
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+Every player must defeat the Level VII Neutral Combat.
+
+\subsection*{\MakeUppercase{Timed Events}}
+\textbf{\nth{2} Round:}
+\begin{itemize}
+  \item Build either Camp \svg{building_special_tent} or Workshop \svg{building_special_gears} for free. %(\svg{bulding_special_gears})
+\end{itemize}
+End of \textbf{\nth{4} Round:}
+\begin{itemize}
+  \item \textbf{one player} must fight a \textbf{Timed Event Fight} of 5 × \svg{bronze}.
```

Comment:
```suggestion
\textbf{End of \nth{4} Round:}
\begin{itemize}
  \item \textbf{One player} must fight a Timed Event Fight of 5~×~\svg{bronze}.
```
I'd suggest using the bold font for the line with round number, like the rest of scenarios do. In the timed event itself, it should be used more sparing. See comparison:

<img width="2062" height="242" alt="image" src="https://github.com/user-attachments/assets/90dbc8a4-985b-4dfc-a837-2692db1f4671" />